### ISSUE TITLE
can't register multiple virtio devices

### ISSUE THREAD

**Issue by thehajime** [lkl/linux#34]
when you added two virtio devices, in a single program like below,

```c
        bs.fd = open("disk.img", O_RDWR);
        lkl_disk_add(bs);

        nd.fd = tap_alloc("tap0");
        lkl_netdev_add(nd, NULL);
```

2nd device is failed to register.

```
[    0.704000] virtio-mmio: Registering device virtio-mmio.0 at 0x27e5010-0x27e5133, IRQ 1.
[    0.704000] virtio-mmio: Registering device virtio-mmio.1 at 0x27e5120-0x27e5243, IRQ 2.
[    0.704000] platform virtio-mmio.1: failed to claim resource 0
[    0.704000] device: `292@0x27e5120:2' invalid for parameter `virtio_mmio.device'
```

**Comment by thehajime** [comment]
i don't know this is the right way or not but with the following patch makes the message above disappeared.

https://gist.github.com/thehajime/2730a4763f02ba1ac417

**Comment by tavip** [comment]
Fix in c25294b0d9d64fe00f05448fd816222c1cae6983
